### DISCUSSION TITLE
Join groups by invite code

### DISCUSSION THREAD

**Discussion by huyuanliang** [Auties00/WhatsappWeb4j#91] [Ideas]
@Auties00 Join groups code by invite ，How do I do it

**Comment by huyuanliang** [comment]
@Auties00 I hope you can help me. I need to use this function in my project. Thank you

**Comment by Auties00** [comment]
> @Auties00 I hope you can help me. I need to use this function in my project. Thank you

Afaim I never implemented that functionality. You can do so by yourself by using the request analyzer though
